### PR TITLE
Test suit for Service Page and its component

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I worked on the servicePage and its component

#
### Type of change :

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

 Frontend: @shahsank3t, @darth-coder00, @Sachin-chaurasiya

### CODE REVIEW

File: catalog-rest-service/src/main/resources/ui/src/components/Modals/AddServiceModal/AddServiceModal.test.tsx [hunk 1]
```diff
@@ -68,59 +66,93 @@ describe('Test AddServiceModal Component', () => {
...
+    expect(database).toBeInTheDocument();
+    expect(driverClass).toBeInTheDocument();
+    expect(description).toBeInTheDocument();
+    expect(ingestionSwitch).toBeInTheDocument();
+    expect(queryByText(container, /day/i)).not.toBeInTheDocument();
+    expect(queryByText(container, /hour/i)).not.toBeInTheDocument();
+    expect(queryByText(container, /minute/i)).not.toBeInTheDocument();
+  });
+
+  it('form with all the fild should render for dashboardServices', async () => {
```
Comment (Sachin-chaurasiya) [inline catalog-rest-service/src/main/resources/ui/src/components/Modals/AddServiceModal/AddServiceModal.test.tsx:101]: The same goes for here.

File: catalog-rest-service/src/main/resources/ui/src/components/Modals/AddServiceModal/AddServiceModal.test.tsx [hunk 2]
```diff
@@ -68,59 +66,93 @@ describe('Test AddServiceModal Component', () => {
...
+    expect(password).toBeInTheDocument();
+    expect(description).toBeInTheDocument();
+    expect(ingestionSwitch).toBeInTheDocument();
+    expect(queryByText(container, /day/i)).not.toBeInTheDocument();
+    expect(queryByText(container, /hour/i)).not.toBeInTheDocument();
+    expect(queryByText(container, /minute/i)).not.toBeInTheDocument();
   });
 
-  it('form with all the fild should render', async () => {
+  it('form with all the fild should render for messagingServices', async () => {
```
Comment (Sachin-chaurasiya) [inline catalog-rest-service/src/main/resources/ui/src/components/Modals/AddServiceModal/AddServiceModal.test.tsx:133]: Here as well.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
Requested few changes otherwise LGTM.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by darth-coder00 — APPROVED** [review]
LGTM

**Pull request merged**